nkms/characters.py

```diff
@@ -9,10 +11,26 @@ class Character(object):
     """
     _server = None
     _server_class = Server
+    _actor_mapping = {}
 
-    def __init__(self, attach_server=True):
+    class ActorNotFound(Exception):
+        """raised when we try to interact with an actor of whom we haven't learned yet."""
+
+    def __init__(self, attach_server=True, crypto_power: CryptoPower=None):
         if attach_server:
             self.attach_server()
+        self._crypto_power = crypto_power or CryptoPower(power_ups=[SigningKeypair])
+
+        class Seal(object):
+            """
+            Can be used
```

Comment:
Hell of a docstring, if I may say so myself.

(obviously this was meant to be longer and clearer.  Next commit.)